apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -222,9 +235,9 @@ suite('Settings >', function() {
       navigator.mozMobileConnections = [{ iccId: 'SIM 1' }, { iccId: null }];
       Settings.init();
 
-      // Four calls for
-      // maxConcatenatedMessages/mmsSizeLimitation/mmsServiceId/smsServiceId
-      sinon.assert.callCount(navigator.mozSettings.createLock, 4);
+      // Five calls for maxConcatenatedMessages/mmsSizeLimitation
+      // sendReadReport/mmsServiceId/smsServiceId/
+      sinon.assert.callCount(navigator.mozSettings.createLock, 5);
```

Comment:
not really sure all these checks with createLock are useful; aren't they implied by the other checks ?